api/src/main/java/org/apache/cloudstack/vm/UnmanagedInstanceTO.java

```diff
@@ -136,6 +137,10 @@ public List<Nic> getNics() {
     public void setNics(List<Nic> nics) {
         this.nics = nics;
     }
+    public String getVncPasswd(){ return vncPasswd; }
+    public void setVncPasswd(String vncPasswd) { this.vncPasswd = vncPasswd; }
```

Comment:
```suggestion
    public String getVncPassword() { 
        return vncPassword;
    }
    
    public void setVncPassword (String vncPasswd) { 
        this.vncPasswd = vncPasswd; 
    }
```

Suggestion to comply with our [coding convention](https://cwiki.apache.org/confluence/display/CLOUDSTACK/Coding+conventions).

Reply:
Ah, makes sense. Done.